src/adapter.js

```diff
@@ -225,6 +225,14 @@ export default class SlackMessageAdapter {
         return true;
       }
 
+      // For options / typeahead, we can return a promise directly (there is no response_url)
+      if (interactionType === 'options') {
```

Comment:
correct me if i'm wrong, but from my reading of this change it seems that even before the change, the code would work the exact same way. i think it would run until the code here: https://github.com/slackapi/node-slack-interactive-messages/blob/a4a1ac12c23f05e8b4a8191b8262553bd734fae3/src/adapter.js#L251-L252

and that would have the exact same side effect.

Reply:
Not quite, since the `callbackResult` is a promise, it actually goes there https://github.com/Startouf/node-slack-interactive-messages/blob/bf19d7b591a159d65a4c08a8e5e07da125342177/src/adapter.js#L253
and triggers a ` callbackResult.then(respond)` which does not work.